scripts/bash_lib.sh

```diff
@@ -210,7 +215,8 @@ rpm_repoquery() {
   set +u
   # return full package list from repository
   if [[ $ID_LIKE =~ ^suse* ]]; then
-    zypper packages -r "${repo_name}" | grep "MariaDB" | awk '{print $8}'
+    sudo zypper --gpg-auto-import-keys packages -r "${repo_name}">/dev/null
```

Comment:
In general, we should try to avoid redirecting to "/dev/null", this makes debug complex and in that case, if the problem is the noisy output probably only redirect SDTOUT, not STDERR.

Reply:
It's actually STOUT that is causing problems. I can skip some number of rows or move the command to the function where MariaDB repo is set up.

Reply:
@fauust 
`zypper --gpg-auto-import-keys refresh mariadb`

in `rpm_setup_mariadb_mirror() `would be a better approach?

Reply:
@fauust I moved the refresh to rpm_setup_mariadb_mirror() and previous version installation goes well.
The command will auto-trust the key. 

` [[ suse opensuse =~ ^suse* ]]
+ sudo zypper --gpg-auto-import-keys refresh mariadb
Looking for gpg key ID C74CD1D8 in cache /var/cache/zypp/pubkeys.
Looking for gpg key ID C74CD1D8 in repository MariaDB.
  gpgkey=https://rpm.mariadb.org/RPM-GPG-KEY-MariaDB

Automatically importing the following key:

  Repository:       MariaDB
  Key Fingerprint:  177F 4010 FE56 CA33 3630 0305 F165 6F24 C74C D1D8
  Key Name:         MariaDB Signing Key <signing-key@mariadb.org>
  Key Algorithm:    RSA 4096
  Key Created:      Wed 30 Mar 2016 05:45:15 PM UTC
  Key Expires:      (does not expire)
  Subkey:           C0F47944DE8F6914 2016-03-30 [does not expire]
  Rpm Name:         gpg-pubkey-c74cd1d8-56fc10ab



    Note: A GPG pubkey is clearly identified by its fingerprint. Do not rely on the key's name. If
    you are not sure whether the presented key is authentic, ask the repository provider or check
    their web site. Many providers maintain a web page showing the fingerprints of the GPG keys they
    are using.
Retrieving repository 'MariaDB' metadata .........................................................................................................................................[done]
Building repository 'MariaDB' cache ..............................................................................................................................................[done]
Specified repositories have been refreshed.
`

Reply:
Then that's even better.

> It's actually STOUT that is causing problems. I can skip some number of rows or move the command to the function where MariaDB repo is set up.

That's why something like `command 1 >/dev/null` would have worked without hiding the error in case there is one.